src/components/tutorial-collection-cards/components/card-heading/types.ts

```diff
@@ -1,5 +1,4 @@
 import type { HeadingProps } from 'components/heading'
-import { CompanyLogoOption } from './company-logo/types'
```

Comment:
Not actually used, all `CompanyLogo` stuff is specific to `collection-card` and was moved there in a previous PR. I just forgot to delete it.